.github/workflows/test-quickstart.yml

```diff
@@ -1,16 +1,12 @@
 name: Test Quickstart
 on:
   workflow_dispatch:
-  # test quickstart changes after merge
   push:
     branches:
       - main
-    paths:
-      - 'quickstart/**'
-  # test quickstart changes before merge
   pull_request:
-    paths:
-      - 'quickstart/**'
+    branches:
```

Comment:
stop waiting for quickstart files to change so this runs earlier in the ziti life cycle